core/src/main/java/org/apache/metamodel/UpdateSummaryBuilder.java

```diff
@@ -1,4 +1,5 @@
 /**
+ * Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Hmm I'm surprise that our current license-check (RAT) plugin doesn't catch this. But definately important to get fixed.